src/protocol.h

```diff
@@ -261,7 +261,16 @@ extern const char* WTXIDRELAY;
 }; // namespace NetMsgType
 
 /* Get a vector of all valid message types (see above) */
```

Comment:
8a1901cc: s/vector/map also in the comments.

Reply:
Fixed.